src/screens/Dashboard/DeparturesWidget.tsx

```diff
@@ -98,11 +98,17 @@ const FavouritesWidget: React.FC = () => {
               : t(DeparturesTexts.message.noFrontpageFavouritesWidget)}
           </ThemeText>
           {new_favourites_info_url && (
-            <Button
```

Comment:
think the button type is not taking the text type as a prop, but changing that right now can affect a lot of places. So this looks good I think 👍

Reply:
As you probably noticed in a later commit, this button was changed into a TouchableOpacity with ThemeText as done another place in the app, so that the text in the button was readable both in light and dark mode 👍

Reply:
yes I did and that looks fine to me :)